### DISCUSSION TITLE
how to fix the header of listview control

### DISCUSSION THREAD

**Discussion by chenkangcrack** [HandyOrg/HandyControl#916] [Q&A] [answered]
dear all,
            I want to know how to fix the position of  the listview control's header.  when i scroll down the listview, the header disappeared as below.

![image](https://user-images.githubusercontent.com/11851538/126619115-bf3ec96c-a2cf-47df-b7c4-570959df3edb.png)
![image](https://user-images.githubusercontent.com/11851538/126619517-0ef7ebeb-2edd-4386-8555-d76ae57fe006.png)

**Comment by NaBian** [comment, marked as answer]
fixed a484306
